web/src/echo.js

```diff
@@ -118,25 +119,7 @@ export function build_display_recipient(message) {
     const display_recipient = emails.map((email) => {
         email = email.trim();
         const person = people.get_by_email(email);
-        if (person === undefined) {
-            // For unknown users, we return a skeleton object.
-            //
-            // This allows us to support zephyr mirroring situations
```

Comment:
I don't know how `zephyr` mirroring works but the commit message indicates that we are not thinking about it.

Reply:
Well, I also don't know about `zephyr` mirroring but what this commit is trying to solve is removing `unknown_local_echo_user` because the person here can never be undefined.